Sources/LaunchItem.swift

```diff
@@ -22,48 +22,69 @@ import Foundation
 
 
 public enum LaunchItem: CustomStringConvertible, Equatable {
-    case RemoteNotificationItem(remoteNotification: RemoteNotification)
-    case LocalNotificationItem(localNotification: UILocalNotification)
-    case OpenURLItem(urlToOpen: URLToOpen)
+    case remoteNotification(item: RemoteNotification)
+    case localNotification(item: UILocalNotification)
+    case openURL(item: URLToOpen)
     @available(iOS 9.0, *)
-    case ShortcutItem(shortcutItem: UIApplicationShortcutItem)
-    case UserActivityItem(userActivity: NSUserActivity)
-    case NoItem
+    case shortcut(item: UIApplicationShortcutItem)
+    case userActivity(item: NSUserActivity)
+    case none
+    
+    // MARK: Equatable
+    
+    public static func ==(lhs: LaunchItem, rhs: LaunchItem) -> Bool {
+        switch (lhs, rhs) {
+        case let (.remoteNotification(itemLHS), .remoteNotification(itemRHS)) where itemLHS == itemRHS:
+            return true
+        case let (.localNotification(itemLHS), .localNotification(itemRHS)) where itemLHS == itemRHS:
+            return true
+        case let (.openURL(itemLHS), .openURL(itemRHS)) where itemLHS == itemRHS:
+            return true
+        case let (.shortcut(itemLHS), .shortcut(itemRHS)) where itemLHS == itemRHS:
+            return true
+        case let (.userActivity(itemLHS), .userActivity(itemRHS)) where itemLHS == itemRHS:
+            return true
+        case (.none, .none):
+            return true
+        default:
+            return false
```

Comment:
it's fine if you compare two enums with different values. it's weird, but I'm not gonna stop you.